pxr/usd/usdSkel/plugInfo.json

```diff
@@ -69,6 +69,14 @@
                         "implementsComputeExtent": true, 
                         "schemaKind": "concreteTyped"
                     }
+                },
+                "Validators": {
+                    "SkelBindingApiAppliedChecker": {
```

Comment:
I think in the 2 validators I introduced in the dev branch, I was trying not to use "Checker" in the name. So

`SkelBindingAPIApplicationValidator` or `SkelBindingAPIApplicationCompliance` ?

cc. @dgovil if you have opinions on this?

Reply:
I did not change this yet, as I'll wait for some agreement (same issue in the MaterialBindingAPI PR). 

My suggestions: (or any combinations of:
 `<APISchema>` + `<Existence | Presence>` + `<Validator | Verifier | Checker>`
 )
- `SkelBindingAPIExistenceVerifier`
- `SkelBindingAPIPresenceValidator`
- ...

`Application` here makes me think of a program / app, rather than being applied, but I'm happy to move forward with any of the options.

Reply:
Since the schemas are an appliedAPI schema, I kind of like  `SkelBindingAPIAppliedValidator`

Reply:
I am fine with that. Will also be renaming the StageMetadataChecker ones I have already synced in to be called StageMetadataValidator. Thanks both.